troupon/account/urls.py

```diff
@@ -1,6 +1,11 @@
-from django.conf.urls import include, url
-from account.views import UserSigninView
+from django.conf.urls import url
+from account.views import UserSigninView, ForgotPasswordView, ResetPasswordView
 
 urlpatterns = [
-    url(r'^signin/$', UserSigninView.as_view()),
+    url(r'^signin/$', UserSigninView.as_view(), name='signin'),
+    url(r'^account/recovery/$', ForgotPasswordView.as_view(), name='account_forgot_password'),
+    url(r'^account/recovery/(?P<recovery_hash>([abcdefghijklmnopqrstuvwxyz0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZ])+)$',
```

Comment:
use `url(r'^account/recovery/(?P<recovery_hash>([a-z0-9A-Z])+)$'` or `\w\d`.
Also remove url prefix `/account`, move to base urls.py so that first route will become `/account/signin`